docs/source/request_state_machine.dot

```diff
@@ -4,11 +4,17 @@ digraph request_state_machine {
     pending -> filled [label = fill];
     pending -> filled [label = try_to_fill];
     pending -> ignored [label = ignore];
+    pending -> claimed [label = try_to_claim];
+
     filled -> claimed [label = try_to_claim];
+
     claimed -> withdrawn [label = withdraw];
     claimed -> l1_resolved [label = resolve];
+
     l1_resolved -> withdrawn [label = withdraw];
+
     filled -> withdrawn [label = withdraw];
     filled -> ignored [label = ignore];
+
```

Comment:
This does not look complete. You added 3 new state transitions in `request.py`, yet here I see only 1 transition added.

Reply:
We currently don't display loop transitions (i.e. state to itself)